cmd/cri-containerd/options/options.go

```diff
@@ -18,38 +18,56 @@ package options
 
 import (
 	"flag"
+	"os"
+	"time"
 
+	"github.com/BurntSushi/toml"
 	"github.com/containerd/containerd"
 	"github.com/spf13/pflag"
 )
 
-// CRIContainerdOptions contains cri-containerd command line options.
-type CRIContainerdOptions struct {
+const (
+	// "default path to the config file"
+	configFilePathArgName = "config-file-path"
+)
+
+//Config contains cri-containerd toml config
+type Config struct {
 	// SocketPath is the path to the socket which cri-containerd serves on.
-	SocketPath string
+	SocketPath string `toml:"socketpath"`
 	// RootDir is the root directory path for managing cri-containerd files
 	// (metadata checkpoint etc.)
-	RootDir string
-	// PrintVersion indicates to print version information of cri-containerd.
-	PrintVersion bool
-	// ContainerdEndpoint is the containerd endpoint path.
-	ContainerdEndpoint string
+	RootDir string `toml:"rootdir"`
 	// ContainerdSnapshotter is the snapshotter used by containerd.
-	ContainerdSnapshotter string
+	ContainerdSnapshotter string `toml:"containerdsnapshotter"`
+	// ContainerdEndpoint is the containerd endpoint path.
+	ContainerdEndpoint string `toml:"containerdendpoint"`
```

Comment:
Can we add a `ContainerdConfig` and include Snapshotter and Endpoint inside?  Those are containerd configurations, and we may add some others into that in the future.